content/applications/hr/referrals.rst

```diff
@@ -562,13 +407,13 @@ Create rewards
 The rewards are the only configurations needed when setting up the Referrals application. Only users
 with :guilabel:`Administrator` rights for the Recruitment application can create or modify rewards.
 To add rewards, navigate to :menuselection:`Referrals app --> Configuration --> Rewards`. Click
-:guilabel:`Create` and a reward form loads. Enter the following information on the form:
+:guilabel:`New` and a reward form loads. Enter the following information on the form:
 
 - :guilabel:`Product Name`: Enter the name as it should appear for the reward.
 - :guilabel:`Cost`: Enter the amount of points required to redeem the reward.
 - :guilabel:`Company`: Using the drop-down menu, select the company the reward will appear for. If a
   reward is to be used for several companies, each company needs to have a record of the reward,
-  with the company listed on the form.
+  with the company listed on the form. This field only appears if in a multi-company environment.
 - :guilabel:`Gift Responsible`: Select the person who will be responsible for procuring and
   delivering the reward to the recipient, using the drop-down menu. This person is alerted when the
   reward is purchased, so they know when to deliver the reward to the recipient.
```

Comment:
just to clarify...are these rewards "purchased" or "earned" via points received?

Reply:
ah, I see - it's earned using the rewards- you have to "spend" the points you earned.

Reply:
I changed it to "boughtin the *Referrals* app" sinc ethe word in the app is "buy" so the nomenclature matches but it is specific to being "bought" IN the app- so I think that clarifies it.